packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -54,9 +54,11 @@ export const getStyles = (props: ICheckboxStyleProps): ICheckboxStyles => {
             },
             ':focus .ms-Checkbox-checkbox': { borderColor: checkboxBorderHoveredColor },
             ':hover .ms-Checkbox-checkmark': {
-              color: checkmarkFontColorHovered,
-              opacity: '1',
               selectors: {
+                [FinePointerSelector]: {

```

Comment:
hmmm. This feels pretty unintuitive. We have a number of places that do something on hover.... highlight a background, show some glyph. Are you suggesting here that all visual treatments wrt hover state should follow this?

Reply:
Yeah, I was trying to talk about this earlier because this won't just be isolated issues here and there, it'll be a widespread problem if mobile is concerned a first class target. In general hover is going to stick so we're going to need some kind of selector that applies only to mouse/stylus clients.

There are alternatives, however. We could do some class based approach like HighContrastSelector for hoverable vs. non-hover clients. But then you get into other issues like not being able to use `:hover` selector directly. There are also more JS-intrusive methods for doing this too.